src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -883,6 +885,231 @@ def cleanse_crash(self,
       return result
 
 
+class AndroidLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """Android libFuzzer runner."""
+  DEVICE_FUZZING_DIR = '/data/fuzz'
+
+  def __init__(self, executable_path, build_directory, default_args=None):
+    """Inits the AndroidLibFuzzerRunner.
+
+    Args:
+      executable_path: Path to the fuzzer executable.
+      build_directory: A MinijailChroot.
+      default_args: Default arguments to always pass to the fuzzer.
+    """
+    super(AndroidLibFuzzerRunner, self).__init__(
+        executable_path=android.adb.get_adb_path(),
+        default_args=self._get_default_args(executable_path, default_args))
+
+    self._copy_local_directory_to_device(build_directory)
+
+  def get_testcase_path(self, log_lines):
+    """Get testcase path from log lines."""
+    path = LibFuzzerCommon.get_testcase_path(self, log_lines)
+    if not path:
+      return path
+
+    return self._get_local_path(path)
+
+  def _get_default_args(self, executable_path, custom_default_args):
```

Comment:
nit: s/custom_default_args/extra_args/ might make it clearer its an addition?

Reply:
done!